### ISSUE TITLE
Detailed and over structured URLs

### ISSUE THREAD

**Issue by christopherbecker** [elad/malver#1]
Hi, this package is rather cool and interesting as to how it works.

However using this package in a real life scenario with a very detailed URL seems to break the script. A URL with params would be a basic example.

Would you consider making it work with these types of URLs?

Thanks.

**Comment by elad** [comment]
You're welcome to submit a PR.

This code was written to prove a point. I came across the "tasearch" domain a couple of years ago while cleaning a friend's malware-infested laptop. Its Chrome installation was fully compromised and forced searches to go through "tasearch." When I visited that website, I noticed it was serving Google ads. I wanted to demonstrate that it's very easy for a company like Google to monitor the domains they serve ads from, because otherwise they are essentially taking part in funding malware and scams.

In other words... I'm not (yet) willing to sink any time into it. :)
